src/components/utils/onfidoApi.js

```diff
@@ -48,7 +48,14 @@ export const requestChallenges = (token, onSuccess, onError) => {
 
 const objectToFormData = (object) => {
   const formData = new FormData()
-  forEach(object, (value, key) => formData.append(key, value))
+  forEach(object, (value, fieldName) => {
+    if (value.blob && value.filename) {
```

Comment:
should check for object type first (`typeof value === "object"`), otherwise it will crash on `undefined`